internal/btf/types.go

```diff
@@ -466,7 +477,7 @@ func inflateRawTypes(rawTypes []rawType, rawStrings stringTable) (namedTypes map
 
 		switch raw.Kind() {
 		case kindInt:
-			typ = &Int{id, name, raw.Size()}
+			typ = &Int{id, name, raw.Size(), IntEncoding((raw.Info & 0x0f000000) >> 24), (raw.Info & 0x00ff0000) >> 16, byte(raw.Info & 0x000000ff)}
```

Comment:
Nit: can you turn the bitmasking into `decodeIntInfo() (IntEncoding, uint32, byte)` or something similar?

Reply:
Done.